types.go

```diff
@@ -156,5 +159,17 @@ type Error struct {
 
 // Error returns a textual description of the test error.
 func (err Error) Error() string {
+	if strings.TrimSpace(err.Body) != "" {
```

Comment:
Style nit, this could be written as a `switch {}`.

Reply:
ok I'll refactor it.